server/db/models/Gig.js

```diff
@@ -10,11 +12,60 @@ module.exports = function (sequelize, DataTypes) {
   }, {
     classMethods: {
 
-      getGigs: function (organizationId) {
+      // accepts object like :
+      // {
+      //   startDate: STRING,
+      //   endDate: STRING ,
+      //   includeStaff: BOOLEAN,
+      //   organizationId: STRING
+      // }
+
+      getGigs: function (info) {
+        var start = moment(info.startDate);
+        var end = moment(info.endDate);
+        var include = [];
+
+        var staffQuery = {
+          model: Gig.associations['Users'].target,
+          attributes: ['id', 'name', 'email'],
+          through: {
+            attributes: [['PositionId', 'id'], 'admin_accepted', 'worker_accepted'],
+            as: 'Position'
+          },
+        };
+
+        var positionQuery = {
+          model: Gig.associations['Positions'].target,
+          attributes: ['title'],
+          through: {
+            attributes: ['required', 'filled'],
+            as: 'statusInfo'
+          }
+        };
+
+        include.push(positionQuery);
+        info.includeStaff && include.push(staffQuery); // just for day view
+
         return Gig.findAll({
           where: {
-            organizationId: organizationId
-          }
+            organizationId: info.organizationId,
+            date: {
+              $between: [start.format('YYYY-MM-DD'), end.format('YYYY-MM-DD')]
+            },
+          },
+          attributes: [
+            'OrganizationId',
+             'title',
+             'type',
+             'complexity',
+             'health',
+             'LocationId',
+             'AttireId',
+             'start_time',
```

Comment:
If we are going to explicitly define the attributes we want back, I think we should be renaming underscore attributes to camelCase to jive with the camelCase style used by react props. Ultimately, Josh and I discussed migrating the database to all camelCase field names.

FYI from the Sequelize Docs:
To rename an attribute, you can pass an array, with two elements - the first is the name of the attribute in the DB (or some kind of expression such as Sequelize.literal, Sequelize.fn and so on), and the second is the name you want the attribute to have in the returned instance